### ISSUE TITLE
Issue when uploading plugins to a new tenant which has "isMaster=false"

### ISSUE THREAD

**Issue by aniketdharia** [adaptlearning/adapt_authoring#1542]
When we try to upload a plugin into a new tenant whose isMaster flag is set to false, the current asset management/plugin upload code looks for the tenant ID and throws and error log that the tenant folder does not exist in the temp directory. Instead it should do a check if the current tenant isMaster and if it is not, it should upload it in its parent tenant. But the parent tenant reference is not maintained anywhere in the DB structure. Need to discuss how to resolve this issue.

This would need to be fixed before releasing the TM feature.

**Comment by taylortom** [comment]
If we're keeping to a single master tenant, then this should just be a case of storing on that?

**Comment by aniketdharia** [comment]
@taylortom yes! but currently it is throwing an error in the console that the tenant folder does not exist
